scripts/qa_crowbarsetup.sh

```diff
@@ -2108,8 +2113,14 @@ function custom_configuration
         ;;
         rabbitmq)
             if [[ $hacloud = 1 ]] ; then
-                proposal_set_value rabbitmq default "['attributes']['rabbitmq']['ha']['storage']['mode']" "'drbd'"
-                proposal_set_value rabbitmq default "['attributes']['rabbitmq']['ha']['storage']['drbd']['size']" "$drbd_rabbitmq_size"
+                if [[ "$want_database_sql_engine" -eq "mysql" ]] ; then
```

Comment:
`$cloudversion` is not good, as we want this for SOC7 as well, and also currently support DRBD for rabbitmq. I know it's not ideal , but `want_database_sql_engine` seems the best one currently